annotations.md

```diff
@@ -0,0 +1,25 @@
+# Annotations
+Several components of the specification, like [Image Manifests](manifest.md) and [Descriptors](descriptor.md), feature an optional annotations property, whose format is common and defined in this section.
+
+This property contains arbitrary metadata.
+
+## Rules
+
+Annotations MUST be a key-value map where both the key and value MUST be strings.
+While the value MUST be present, it MAY be an empty string.
+Keys MUST be unique within this map, and best practice is to namespace the keys.
+Keys SHOULD be named using a reverse domain notation - e.g. `com.example.myKey`.
+Keys using the `org.opencontainers` namespace are reserved and MUST NOT be used by other specifications and extensions.
+If there are no annotations then this property MUST either be absent or be an empty map.
+Implementations that are reading/processing manifest lists MUST NOT generate an error if they encounter an unknown annotation key.
+
+See [Pre-Defined Annotation Keys](manifest.md#pre-defined-annotation-keys).
```

Comment:
You removed this `manifest.md` section, and the `annotations.md` section if right after this line.  So I think we should drop this line.